tests/testthat/test-sample_struct.R

```diff
@@ -157,7 +157,7 @@ test_that("get_full_sample_struct works", {
   # reference to compare against
   full_spl_str_ref <- list(
     catch = data.frame(Yr = 101:110, Seas = 1, FltSvy = 1, SE = 0.005),
-    CPUE = data.frame(Yr = c(105, 110), Seas = 7, FltSvy = 2, SE = 0.2),
+    CPUE = data.frame(Yr = c(105, 110), Seas = 7, FltSvy= 2, SE = 0.2),
```

Comment:
Not sure why this line changed but it should be changed back.

Reply:
Yep, that one was a mistake! I think the styler should take care of it after merging in, though.